Packs/CommonTypes/IncidentFields/incidentfield-Custom_Query_Results.json

```diff
@@ -0,0 +1,27 @@
+{
+    "associatedToAll": true,
```

Comment:
Please rethink whether all these fields need to be associated to all. They can be in common pack and still associated to your specific pack. This field for example doesn't belong in Phishing incidents.